src/preprocessor.ts

```diff
@@ -92,6 +87,14 @@ export function process(
   return modified;
 }
 
+/**
+ * This is the function Jest uses to check if it has the file already in cache
+ * @param {string} fileData
+ * @param {Path} filePath
+ * @param {string} jestConfigStr
+ * @param {TransformOptions} transformOptions
+ * @returns {string}
+ */
```

Comment:
This looks nice but do we really need it given that the function declaration visibly presents this information? The concern is keeping the two in sync - it's only a minor issue but it seems avoidable

Reply:
No you're right. I'll just leave the commenmt